ton_client/client/src/api/dispatch.rs

```diff
@@ -332,18 +249,13 @@ where
     R: Send + Serialize,
     F: Fn(Arc<ClientContext>, P) -> ApiResult<R>,
 {
-    fn handle(&self, context: Arc<ClientContext>, params_json: &str) -> JsonResponse {
+    fn handle(&self, context: Arc<ClientContext>, params_json: &str) -> ApiResult<String> {
         match parse_params(params_json) {
             Ok(params) => {
                 let result = (self.handler)(context, params);
-                match result {
-                    Ok(result) => {
-                        JsonResponse::from_result(serde_json::to_string(&result).unwrap())
-                    }
-                    Err(err) => JsonResponse::from_error(err),
-                }
+                result.map(|x| serde_json::to_string(&x).unwrap())
```

Comment:
Use `CANNOT_SERIALIZE_RESULT` or `cannot_serialize_result` instead of `unwrap`

Reply:
unwrap removed